package.json

```diff
@@ -15,34 +15,27 @@
   "dependencies": {
     "boxen": "^4.2.0",
     "chalk": "^4.1.0",
```

Comment:
Am I right in thinking we technically don't need to have chalk as our own dep, given it's in chalk-pipe? (and was in inquirer-chalk-pipe)